modules/tinymce/CHANGELOG.md

```diff
@@ -13,7 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `setText` and `setIcon` methods added to menu button and toolbar button apis. #TINY-9268
 - New `highlight_on_focus` option which enables highlighting the content area on focus. #TINY-9277
 - New `fontsizeinput` toolbar item which allows the user to set the size via input and also increase and decrease it with `+` and `-` buttons. #TINY-9429
-
+- Added `skipFocus` option to the `ToggleToolbarDrawer` command to preserve focus. #TINY-9337
+- Added typography plugin to the default `menubar`. #TINY-9450
```

Comment:
We generally avoid talking about premium plugins in the open source changelog
https://github.com/tinymce/tinymce/pull/8342#discussion_r1050257925

Reply:
Removed `- Added typography plugin to the default `menubar`. #TINY-9450` and `- Fixed the space between a comma and a quote in typography. #TINY-9442`